editor/src/clj/editor/gui.clj

```diff
@@ -1209,10 +1205,13 @@
                                                     (dissoc :original-value))))))
                                 prop-kw->prop-info)))))))
   (input child-build-errors g/Any :array)
-  (output build-errors-gui-node g/Any (g/fnk [_node-id id id-counts layer layer-names]
-                                        (g/package-errors _node-id
-                                                          (prop-unique-id-error _node-id :id id id-counts "Id")
-                                                          (validate-layer false _node-id layer-names layer))))
+  (output build-errors-gui-node g/Any
+          (g/fnk [_node-id basic-gui-scene-info id id-counts layer]
+            (let [layer-names (:layer-names basic-gui-scene-info)]
+              (g/package-errors
+                _node-id
+                (prop-unique-id-error _node-id :id id id-counts "Id")
+                (validate-layer false _node-id layer-names layer)))))
```

Comment:
Get `layer-names` from `basic-gui-scene-info`. Also decreased the indentation for readability.